async-nats/src/lib.rs

```diff
@@ -445,7 +447,7 @@ impl ConnectionHandler {
                 length,
             } => {
                 if let Some(subscription) = self.subscriptions.get_mut(&sid) {
-                    let message = Message {
+                    let message: Message = Message {
```

Comment:
```suggestion
                    let message = Message {
```
This change might not be needed (most probably added by me, during rebase...).